R/bioconductor.R

```diff
@@ -8,24 +10,18 @@ renv_bioconductor_init <- function(library = NULL) {
 
 renv_bioconductor_init_biocmanager <- function(library = NULL) {
 
-  if (renv_package_checking())
```

Comment:
This was possibly ill-advised, but I needed it to get the tests working, and it doesn't seem to cause problems?

Reply:
This might have been because I was worried about accidentally installing BiocManager when on CRAN?

Reply:
Yeah, makes sense. I think we should protected from that other ways now.